src/Symfony/Component/Workflow/Event/Event.php

```diff
@@ -75,6 +75,9 @@ public function getWorkflowName()
         return $this->workflow->getName();
     }
 
+    /**
+     * @return string|null
```

Comment:
all those places returning a metadata should use `mixed` as well, as they are proxies for the store method where I asked you to change the type